docs/Reference.md

```diff
@@ -253,18 +253,19 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 ###### Formats
 
-| Type                     | Structure           | Example                       |
-| ------------------------ | ------------------- | ----------------------------- |
-| User                     | <@USER_ID>          | <@80351110224678912>          |
-| User \*                  | <@!USER_ID>         | <@!80351110224678912>         |
-| Channel                  | <#CHANNEL_ID>       | <#103735883630395392>         |
-| Role                     | <@&ROLE_ID>         | <@&165511591545143296>        |
-| Slash Command \*\*       | </NAME:COMMAND_ID>  | </airhorn:816437322781949972> |
-| Standard Emoji           | Unicode Characters  | 💯                            |
-| Custom Emoji             | <:NAME:ID>          | <:mmLol:216154654256398347>   |
-| Custom Emoji (Animated)  | <a:NAME:ID>         | <a:b1nzy:392938283556143104>  |
-| Unix Timestamp           | <t:TIMESTAMP>       | <t:1618953630>                |
-| Unix Timestamp (Styled)  | <t:TIMESTAMP:STYLE> | <t:1618953630:d>              |
+| Type                    | Structure           | Example                       |
+|-------------------------|---------------------|-------------------------------|
+| User                    | <@USER_ID>          | <@80351110224678912>          |
+| User \*                 | <@!USER_ID>         | <@!80351110224678912>         |
+| Channel                 | <#CHANNEL_ID>       | <#103735883630395392>         |
+| Role                    | <@&ROLE_ID>         | <@&165511591545143296>        |
+| Slash Command \*\*      | </NAME:COMMAND_ID>  | </airhorn:816437322781949972> |
+| Standard Emoji          | Unicode Characters  | 💯                            |
+| Custom Emoji            | <:NAME:ID>          | <:mmLol:216154654256398347>   |
+| Custom Emoji (Animated) | <a:NAME:ID>         | <a:b1nzy:392938283556143104>  |
+| Unix Timestamp          | <t:TIMESTAMP>       | <t:1618953630>                |
+| Unix Timestamp (Styled) | <t:TIMESTAMP:STYLE> | <t:1618953630:d>              |
+| Guild Navigation        | <id:TYPE>           | <id:customize>                |
```

Comment:
Also, I don't know if it's just me but both the `<id:TYPE>` as well as the example are missing the `<` / `>`  in the web view file preview.

Reply:
You can always use `\<` to prevent it

Reply:
It's fixed

Reply:
> You can always use `\<` to prevent it

That would've been my approach as well but I wasn't too sure why / if that issue would've even rendered like this on the docs...